src/services/map-events-mananger/consts/pickOptions.enum.ts

```diff
@@ -3,10 +3,12 @@
  *  PICK_FIRST  - first entity will be picked . use Cesium.scene.pick()
  *  PICK_ONE    - in case a few entities are picked plonter is resolved . use Cesium.scene.drillPick()
  *  PICK_ALL    - all entities are picked. use Cesium.scene.drillPick()
+ *  MULTI_PICK  - every selected entity is picked. If chosen twice will be unselected. use Cesium.scene.pick()
```

Comment:
After we discussed about it we decided that multi selection should be in a separate service, . That because we need to keep separation of concerns , Map eventManager should be responsible of map event only. hoarding entities per event type for multi selection is an other concern.  of course MultiSelection selection will use the MapEventManager .   
please refactor it to MultiSelectionService